Makefile

```diff
@@ -14,6 +14,9 @@
 
 GO ?= go
 
+export GOPROXY=https://proxy.golang.org
+export GOSUMDB=https://sum.golang.org
```

Comment:
Could we only set this in travis config file?

Reply:
Hm, I think we could also speed up local development with this. Adding it to the Makefile and script would keep things consistent between travis and my local working env. WDYT?

Reply:
Okay, I add them the travis only.